src/main/java/com/example/learningspring/model/Book.java

```diff
@@ -14,13 +14,13 @@
   @NotEmpty(message = "Author Name can not be empty")
   private String authorName;
   private String description;
-  private Boolean published;
+  private String published;
```

Comment:
Why this column is changed to String?

Reply:
just need to show the display like **active** or **in-active**